fe-ionic/Jenkinsfile

```diff
@@ -22,6 +22,7 @@ odsQuickstarterPipeline(
 
   stage("update ionic cli") {
     sh "npm install -g ionic@${ionicVersion} && ionic version"
+    sh "npm install -g eslint && eslint -v"
```

Comment:
I think lately we have tried to pin the version to avoid running into issues sometime after the quickstarter installation. Maybe we can do that for `eslint` as well?

That said, `ionic@${ionicVersion}` apparently hasn't had the desired effect since now things broke. Can we double check why this is happening?

Reply:
I will try to find when did they removed the tslint from the configuration. Also, I will add the eslint version tag in the PR.

Reply:
Ionic in version 5 has removed tslint as a default linter option. It starts without any linter so the user can choose which one will they use. Ours worked because we had tslint added in package.json, and it was all ok until the line in init.sh that actually changes tslint.json was added. This change triggered the bug. I have removed that line as it is not needed in eslint and I have added eslint as a default option for linter.